test/testvm.py

```diff
@@ -611,6 +611,9 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
         if tty:
             return subprocess.Popen(cmd)
 
+        quoted = " ".join("'%s'" % x for x in cmd)
+        cmd = [ "/bin/sh", "-x", "-c", quoted + " | tee /tmp/qemup.log" ]
```

Comment:
What about giving this a unique name and leaving it in the current directory?  Otherwise it will be overwritten by other test runs.